main.go

```diff
@@ -182,18 +181,11 @@ func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 	name := mux.Vars(r)["name"]
 	plan := r.FormValue("plan")
 
-	helmTargetArtifactTypeValue := "Helm"
-	yamlsTargetArtifactTypeValue := "Yamls"
-	knativeTargetArtifactTypeValue := "Knative"
-
 	re := regexp.MustCompile(`artifacttype:\\s.*`)
 	artifactTypematch := re.FindString(plan)
-	artifactType := yamlsTargetArtifactTypeValue
-	if strings.HasSuffix(artifactTypematch, knativeTargetArtifactTypeValue) {
-		artifactType = knativeTargetArtifactTypeValue
-	} else if strings.HasSuffix(artifactTypematch, helmTargetArtifactTypeValue) {
-		artifactType = helmTargetArtifactTypeValue
-	}
+	match := []rune(artifactTypematch)
```

Comment:
Before this we could possibly check if the match was successful or empty. If empty we can use s hard-coded empty vslue like "".

Reply:
Sure, you are right, there should be a check to see if the match is successful or not. So if it's empty, you want the artifactType to be set to "", is that correct?

Reply:
Yes